.github/workflows/python_linting.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: flake8 Linting
+
+on:
+  push:
+    paths:
+      - '**.py'
+
+jobs:
+  flake8_py3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+          architecture: x64
+      - name: Install flake8
+        run: pip install flake8
+      - name: Run flake8
+        uses: suo/flake8-github-action@releases/v1
+        with:
+          checkName: flake8_py3  # NOTE: this needs to be the same as the job name
```

Comment:
Could these tests be brought into the Makefile, that way they run consistently on local dev and ci?
See https://github.com/alphagov/cyber-security-api/blob/master/.github/workflows/pythonpackage.yml#L31

Reply:
Sure can.

Reply:
nice!